### PR TITLE
Add Session Manager Plugin for the AWS CLI version 1.0.37.0

### PR DESCRIPTION
**Pull request by nathanielvarona**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by seanscottking** [comment]
Thanks @nathanielvarona 👍

Team, please merge!

**Comment by anbotero** [comment]
I’m all for this! I was about to look into it. I’m not sure about the version, since the package it downloads it says **latest** all the time, so it might change over time but not necessarily the brew `version`.

File: Casks/session-manager-plugin.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'session-manager-plugin' do
+  version '1.0.37.0'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/session-manager-plugin.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/session-manager-plugin.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'session-manager-plugin' do
+  version '1.0.37.0'
```
Comment (vitorgalvao) [inline Casks/session-manager-plugin.rb]: Seeing as `url` doesn’t point to a specific version and there’s no [appcast](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md), this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/homebrew/homebrew-cask/blob/37f8d181d1789cbb06c6e73625587fdbe77ca225/Casks/kibako.rb#L2L3).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Is this open-source?

**Comment by vitorgalvao** [comment]
Closing for lack of reply.

**Pull request closed without merging**